content/battery.md

```diff
@@ -19,167 +19,155 @@ section: hardware
 tableOfContents: true
 ---
 
-Every running program consumes the battery. This could be a program that is part of the operating system, or a program currently in use like <u>Firefox</u> or <u>Libre Office</u>. We recommend using [TLP](http://linrunner.de/en/tlp/tlp.html) to quickly reduce overall power consumption and using [powertop](https://01.org/powertop) to check what software is consuming the battery.
+Pop!\_OS does a lot to conserve the battery life of your laptop just by default, but there's additional things to be aware of that can stretch that battery life even longer. Choosing the right options to match your work or play style can help keep you away from the wall socket.
 
-The biggest consumer of power is the display backlight. Up to 10% more battery life can be gained by reducing display brightness. Our laptops don't currently have an ambient light sensor and brightness needs manual adjustment with <kbd>Fn</kbd>+<kbd>F8</kbd> and <kbd>Fn</kbd>+<kbd>F9</kbd>.
+## Screen brightness
```

Comment:
Make sure all titles are in title case. There are a few instances of this throughout the article.

Reply:
Done. I also adjusted the titles so they're a bit more consistent. Will be updated on the next git push.